vms/platformvm/vm.go

```diff
@@ -94,6 +94,7 @@ var (
 
 // VM implements the snowman.ChainVM interface
 type VM struct {
+	metrics
```

Comment:
I was copying the `avm` style but it looks like I borked a test: https://github.com/ava-labs/avalanchego/blob/084300fa306c21c177cd699a1e55aa3363d8b56f/vms/avm/vm.go#L69

Reply:
```
=== RUN   TestUptimeReporting
    vm_test.go:2583: duplicate metrics collector registration attempted
```